store/src/main/java/org/apache/rocketmq/store/ConsumeQueue.java

```diff
@@ -84,6 +84,59 @@ public boolean load() {
         return result;
     }
 
+    public void checkCommitLogAndConsumeQueueConsistent() {
+
+        String queueDir = this.storePath
+            + File.separator + topic
+            + File.separator + queueId + File.separator;
+
+        long maxOffsetInQueue = getMaxOffsetInQueue();
```

Comment:
I would like to use `long lastRecordOffset = getMaxOffsetInQueue();` instead, to avoid using too many `+1 or -1` in below code block. Actually, `getMaxOffsetInQueue()` returns the next write position, not the position of last consume queue record.

Reply:
refactor